.github/ISSUE_TEMPLATE/enhancement_request.yml

```diff
@@ -0,0 +1,49 @@
+name: Enhancement Request
+description: Suggest an enhancement for this project.
+labels: [enhancement, triage]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Thank you for filing an enhancement request!**
+
+  - type: dropdown
+    id: component
+    attributes:
+      label: Related Component
+      description: Which component does this relate to?
+      options:
+        - compiler
+        - forc
+        - forc-lsp
+        - forc-fmt
+        - another forc plugin
+        - other
```

Comment:
we may want to add forc-deploy as a separate option here as it is highly used.

Reply:
added